### DISCUSSION TITLE
Bindings with zsh

### DISCUSSION THREAD

**Discussion by PitiBouchon** [raphamorim/rio#388] [Q&A]
I want to bind CTRL+left to moving backward a word, so I have in my `.zshrc : bindkey "^[[1;5D" backward-word`
It works fine in the default ubuntu terminal but not in rioterm where it prints a D in the terminal
So I tried :
```
[bindings]
keys = [
  { key = "left", with = "control", bytes = [27, 91, 49, 59, 53, 67] }
]
```

and also with `action = "ReceiveChar"` but none of them works

**Comment by MrPandir** [comment]
+1

@raphamorim could you please add a way to use the `CSI` format instead of `bytes`. Or explain how to convert from `CSI` to `bytes`, preferably it was in the documentation.
